packages/calcite-components/src/components/option/option.tsx

```diff
@@ -59,8 +59,7 @@ export class Option {
   //
   //--------------------------------------------------------------------------
 
-  @Element()
-  private el: HTMLCalciteOptionElement;
+  @Element() private el: HTMLCalciteOptionElement;
```

Comment:
This formatting-only change is unrelated to a move/dropping of Element section.

Could you remove those from this PR? We can do a separate PR with linting fixes.

Applies to other files with only lint fixes.